dask/bag/tests/test_bag.py

```diff
@@ -299,6 +299,15 @@ def test_map_partitions():
     assert b.map_partitions(len).name == b.map_partitions(len).name
     assert b.map_partitions(lambda a: len(a) + 1).name != b.map_partitions(len).name
 
+def test_map_partitions_args():
+    from dask.imperative import value
+    assert list(
+        b.map_partitions(lambda a, m: [x * m for x in a], 2)
+         .map_partitions(sum)) == [20, 20, 20]
+    assert list(
+        b.map_partitions(lambda a, m: [x * m for x in a], value(2))
+         .map_partitions(sum)) == [20, 20, 20]
```

Comment:
If we're going to support `*args` then it would be also good to check multiple extra arguments.